### PR TITLE
Add unsubscribe interactionType

### PR DESCRIPTION
**Pull request by guayang-adobe**
@sawhneyh @prabhum2 Please review

### CODE REVIEW

File: extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json [hunk 1]
```diff
@@ -21,10 +21,11 @@
           "title": "Interaction Type",
           "type": "string",
           "description": "Specify action made by user on a message.",
-          "enum": ["click", "open", "action"],
+          "enum": ["click", "open", "action", "unsubscribe"],
```
Comment (sawhneyh) [inline extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json:24]: We can remove "action" from the enum list as is not getting used.

Comment (guayang-adobe) [reply extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json]: @sawhneyh I have removed "action" now

**Comment by guayang-adobe** [comment]
@sawhneyh @prabhum2 Please help approve and merge. Thanks.

**Review by sawhneyh — APPROVED** [review]

**Comment by guayang-adobe** [comment]
Thank you @sawhneyh for reviewing and approving. @prabhum2 Can you help merge this?

File: extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json [hunk 2]
```diff
@@ -21,10 +21,11 @@
           "title": "Interaction Type",
           "type": "string",
           "description": "Specify action made by user on a message.",
-          "enum": ["click", "open", "action"],
+          "enum": ["click", "open", "unsubscribe"],
           "meta:enum": {
             "click": "Link in a message is clicked by the user.",
-            "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application."
+            "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application.",
```
Comment (prabhum2) [inline extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json:27]: @guayang-adobe - Please be aware that meta:enum values are displayed as drop downs  in ansible UI mainly used for segmentation or filtering. short description like below is recommended for them to be displayed properly in UI. 
“click”: “Message link clicked.“,
“open”: “Message opened.“,
“unsubscribe”: “Unsubscribed from push messages”

**Review by prabhum2 — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**